### ISSUE TITLE
Hard-coded `/home/user`

### ISSUE THREAD

**Issue by sjvudp** [QubesOS/qubes-issues#5633]
**Qubes OS version**
R4.0.3

**Affected component(s) or functionality**
_Filled-in later._

**Brief summary**
(I'm new to Qubes OS, and I didn't have networking when I hit the error, now writing with Windows, so I'll provide more details once I have a working Qubes OS with networking.)
When trying to upgrade a VM (I think it was `whonix`), the procedure failed due to a non-existing `/home/user`.

**To Reproduce**
_Steps to reproduce the behavior: (filled in later)_
_1. Go to '...'_
_2. Click on '....'_
_3. Scroll down to '....'_
_4. See error_

**Expected behavior**
There shouldn't be such an error message.

**Actual behavior**
Error appears, and there's nothing the user could do to prevent it.

**Screenshots**
_If applicable, add screenshots to help explain your problem._

**Additional context**
_Add any other context about the problem here._

**Solutions you've tried**
_If applicable, any solutions or workarounds you've already tried._

**Relevant [documentation](https://www.qubes-os.org/doc/) you've consulted**
_A list of links to the Qubes documentation (or other relevant software documentation) pages you have already consulted._

**Related, [non-duplicate](https://www.qubes-os.org/doc/reporting-bugs/#new-issues-should-not-be-duplicates-of-existing-issues) issues**
Debian-8 template has no /home/user directory #3728

**Comment by andrewdavidwong** [comment]
> When trying to upgrade a VM (I think it was whonix), the procedure failed due to a non-existing /home/user.

It's very important to know which Template exhibited this problem. Can you confirm whether it was Whonix and, if so, which version?

**Comment by sjvudp** [comment]
First thanks for your patience: I had been installing Qubes OS on a 64GB USB stick, and it seems that's part of the problem:
When writing a lot of data with Linux, the buffers become dirty and there are no buffers available for reading, so you get read stalls for several seconds. That, in turn, seems to cause various timeouts when Qubes Dom0 tried to "talk" to the VMs during installation. Unfortunately those errors are not reported. I did install two times on the stick, but I was unable to log into  the whonix VM, and the other VMs refused to start after some, point, too...
So today I installed QubesOS on an external hard drive (which is not that handy), and the outcome is quite different: All VMs behave as expected. 
Sorry for the noise!
